src/ecalc/libraries/neqsim/neqsim_ecalc_wrapper/thermo.py

```diff
@@ -30,6 +31,35 @@ class NeqsimEoSModelType(Enum):
     GERG_PR = neqsim.thermo.system.SystemPrEos
 
 
+class NeqsimFluidComponent(BaseModel):
+    """
+    A representation of a NeqSim Fluid Component. As a part of a composition. A
+    gas or fluid, its amount and unit.
+    """
+    name: str
+    total: float
```

Comment:
```suggestion
    total_fraction: float
```

Reply:
this was to have a 1:1 relation to the names given in neqsim (reuse of names), so I dont know what is correct. you know this?